Sprint_Tests/sofia_zahoruiko.md

```diff
@@ -0,0 +1,18 @@
+1. He had a deep understanding of the layout and working with advanced technologies at the frontend.
+After a short break of several years and a deep depression, plus listening to heavy music, he has poorly understanding of the modern trends and drinks a lot.
+If he could handle depression after his wife and kid death, he wouldn't have to find a job with a relevant salary now.
+
```

Comment:
He used to be good at markup and work with the top notch front-end technologies
`on the front-end` - wrong preposition
`he has poor understanding` - wrong word form
If he had coped with/ overcame the depression after his wife and son's deaths he wouldn't have an issue with finding a job with relevant salary now. - this is mixed conditional